### PR TITLE
Contributing index - Fixes for grammar and clarity

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Fixes for grammar and clarity

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/contributing/index.md [hunk 1]
```diff
@@ -5,14 +5,18 @@ description: Learn how to provide feedback and contribute to Sumo Logic Docs, an
...
+We're a community of developers and practitioners dedicated to building reliable and secure modern apps. At Sumo Logic, we believe in the power of collaboration and community-driven innovation. Our open-source projects go beyond code—they're about fostering a vibrant, inclusive community where every contribution, big or small, makes an impact.
 
-There are many ways to contribute to Sumo Docs, beyond writing code. Here's how you can get involved.
+Whether you’re fixing bugs, adding features, writing documentation, or sharing feedback, your contributions help others to learn and grow within the Sumo Logic ecosystem. From typo fixes to new docs, we welcome all forms of participation.
+
+Our open-source [documentation](https://github.com/SumoLogic/sumologic-documentation) project is hosted on GitHub, where you can find all the resources to get started — guidelines on creating files, writing in Markdown, using templates, and submitting pull requests.
```
Comment (jpipkin1) [inline docs/contributing/index.md:15]:
```suggestion
Our open-source [documentation](https://github.com/SumoLogic/sumologic-documentation) project is hosted on GitHub, where you can find all the resources to get started—guidelines on creating files, writing in Markdown, using templates, and submitting pull requests.
```

File: docs/contributing/index.md [hunk 2]
```diff
@@ -5,14 +5,18 @@ description: Learn how to provide feedback and contribute to Sumo Logic Docs, an
...
+
+Whether you’re a seasoned developer or just starting, join us in building tools that help everyone better understand and optimize their systems. Ready to contribute? Explore our [contributing guide](https://help.sumologic.com/docs/contributing/) and get involved today!
+
+You can also explore more Sumo Logic open-source projects on our [GitHub repository](https://github.com/SumoLogic).
```
Comment (jpipkin1) [inline docs/contributing/index.md:19]:
```suggestion
You can also explore more Sumo Logic open-source projects on our [GitHub site](https://github.com/SumoLogic).
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce! I have just a couple of minor suggestions.

File: docs/contributing/index.md (hunk 2 above)
Comment (kimsauce) [reply docs/contributing/index.md:19]: great catch

**Pull request merged**